shared/react-native/ios/Keybase/AppDelegate.m

```diff
@@ -22,6 +22,20 @@ @interface AppDelegate ()
 
 @implementation AppDelegate
 
+- (BOOL)addSkipBackupAttributeToItemAtPath:(NSString *) filePathString
+{
+  NSURL* URL= [NSURL fileURLWithPath: filePathString];
```

Comment:
If you want, try to make the obj consistent with rest of file? There are extra spaces for args.